### DISCUSSION TITLE
Monorepo Setup using pnpm

### DISCUSSION THREAD

**Discussion by artur-dani** [cawa-93/vite-electron-builder#946] [Q&A]
Firstly, thank you for sharing this amazing boilerplate. 

I would like to use this boilerplate in a monorepo project that would allow to share UI components between Electron and Web app, and potentially add a NestJS backend server. I was thinking of using pnpm workspaces for that, and adding storybook would be a plus too. 

I'd love to know how can this be achieved using this boilerplate or is there any variant already implementing it.

Cheers

**Comment by adnanlah** [comment]
The template architecture has been restructured to be an npm monorepo itself which is really incredible if you think about it. You can change it to be an pnpm monorepo easily.

You can read more about it here: https://github.com/cawa-93/vite-electron-builder/discussions/1014
